cpu/native/Makefile.dep

```diff
@@ -21,6 +21,10 @@ ifneq (,$(filter periph_rtc,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
+ifneq (,$(filter can_linux,$(USEMODULE)))
+  LINKFLAGS += -lsocketcan
+endif
```

Comment:
Normally, this should go to `cpu/native/Makefile.include`, but I won't insist.

Reply:
I will move that, but I think we should get this into master, to get the seperation in and the majority of this cleanup done, i think there will be future smaller patches that move such 3 line chunks at even more correct places.